CONTRIBUTING.md

```diff
@@ -184,15 +184,15 @@ core modules.
 ### Step 6: Push
 
 ```text
-$ git push origin my-feature-branch
+$ git push origin my-branch
 ```
 
-Go to https://github.com/yourusername/node and select your feature branch.
+Go to https://github.com/yourusername/node and select your branch.
 Click the 'Pull Request' button and fill out the form.
 
 Pull requests are usually reviewed within a few days. If there are comments
 to address, apply your changes in a separate commit and push that to your
-feature branch. Post a comment in the pull request afterwards; GitHub does
+branch. Post a comment in the pull request afterwards; GitHub does
 not send out notifications when you add commits.
```

Comment:
We should reword this part now that GitHub does send notifications about new commits. That can be done in another PR, I just thinking out loud.